src/wasm-binary.h

```diff
@@ -390,8 +390,8 @@ class WasmBinaryWriter : public WasmVisitor<WasmBinaryWriter, void> {
   void writeMemory() {
     if (wasm->memory.max == 0) return;
     if (debug) std::cerr << "== writeMemory" << std::endl;
-    o << int8_t(BinaryConsts::Memory) << int8_t(log2(wasm->memory.initial))
-                                      << int8_t(log2(wasm->memory.max))
+    o << int8_t(BinaryConsts::Memory) << int8_t(ceil(log2(wasm->memory.initial)))
+                                      << int8_t(ceil(log2(wasm->memory.max)))
```

Comment:
For example, `500` bytes of memory would be `8.97` (`log2` gets an integer cast to a float, returns a float). With `ceil` we get `9`. Then in the round trip, we get `512`, a little more. But better than `256` which breaks some spec tests.

`int8_t` is from the v8 binary format.